targets/targets.json

```
@@ -5945,6 +5943,19 @@
         },
         "detect_code": [
             "1302"
+        ],
+        "supported_c_libs": {
+            "arm": [
+                "std",
+                "small"
+            ],
+            "gcc_arm": [
+                "std",
+                "small"
+            ]
+        },
+        "supported_application_profiles": [
+            "bare-metal"
```

```suggestion
            "bare-metal"
```
Can you not build with the full profile?

---

You should be able to, I'm working on changing this and will confirm once I have tested a full build. 👍🏿